README.md

```diff
@@ -25,17 +25,18 @@ Usage: dotnet-project-licenses [options]
 | `-mapping, --licenseurl-to-license-mappings`  | When used, this option allows to override the url to license mapping built into the application (see [here](src/NuGetUtility/LicenseValidator/UrlToLicenseMapping.cs))                                                                                                                                                                                                                                                                                     |
 | `-override, --override-package-information`   | When used, this option allows to override the package information used for the validation. This makes sure that no attempt is made to get the associated information about the package from the available web resources. This is useful for packages that e.g. provide a license file as part of the nuget package which (at the time of writing) cannot be used for validation and thus requires the package's information to be provided by this option. |
 | `-d, --license-information-download-location` | When used, this option downloads the html content of the license URL to the specified folder. This is done for all NuGet packages that specify a license URL instead of providing the license expression.                                                                                                                                                                                                                                                  |
 | `-o, --output`                                | This Parameter accepts the value `table`, `json` or `jsonPretty`. It allows to select the type of output that should be given. If omitted, the output is given in tabular form.                                                                                                                                                                                                                                                                            |
 | `-err, --error-only`                          | This flag allows to print only packages that contain validation errors (if there are any). This allows the user to focus on errors instead of having to deal with many properly validated licenses.                                                                                                                                                                                                                                                        |
+| `-ii, --include-ignored`                       | This flag allows to explicitely include ignored packages in the output. licenses.                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Thanks a lot for this nice clean PR!
Maybe move to line 34?
"This flag allows to explicitely include ignored packages in the output. licenses." -> "This flag allows to **explicitly** include packages with ignored licenses in the output."

As you know, I think this should have been default true but i'm very glad there is an option. Lets agree to disagree ;)